docs/DeveloperGuide.md

```diff
@@ -615,19 +618,19 @@ Others:
 
 ### Glossary
 
-* **Mainstream OS**: Windows, Linux, Unix, OS-X.
-* **Student**: A student in the database, identified by their name and NUSNET ID. Each student can be in multiple groups, and can have scores for multiple assessments.
-* **Group**: A group of students, identified by the group name.
-* **Assessment**: An assessment is identified by the assessment name.
-* **Score**: The score that a student has attained for an assessment, stored in percentage and can be recorded up to 2 decimal places. Each student can only have 1 score per assessment.
-* **Student list**: The list of students displayed on the right panel of Source Control. Student list can be filtered to display selected students only.
-* **Flag**: Arguments flags are used to indicate different types of user inputs e.g. `-n` for student name, and `-g` for group. More about flags can be found [here](https://ay2122s1-cs2103t-w08-2.github.io/tp/UserGuide.html#glossary).
+* **Mainstream OS:** Windows, Linux, Unix, OS-X.
+* **Student:** A student in the database, identified by their name and NUSNET ID. Each student can be in multiple groups, and can have scores for multiple assessments.
+* **Group:** A group of students, identified by the group name.
+* **Assessment:** An assessment is identified by the assessment name.
+* **Score:** The score that a student has attained for an assessment, stored in percentage and can be recorded up to 2 decimal places. Each student can only have 1 score per assessment.
+* **Student list:** The list of students displayed on the right panel of Source Control. Student list can be filtered to display selected students only.
+* **Flag:** Arguments flags are used to indicate different types of user inputs e.g. `-n` for student name, and `-g` for group. More about flags can be found [here](https://ay2122s1-cs2103t-w08-2.github.io/tp/UserGuide.html#glossary).
 
 <br>
 
 --------------------------------------------------------------------------------------------------------------------
 
-## **Appendix: Instructions for manual testing**
+## **Appendix: Instructions for Manual Testing**
```

Comment:
I think it's quite inconsistent that you un-titlecased most of the titles, but made this one titlecase. I think everything being in titlecase would make sense

Reply:
Hm my intention is only capitalizing the 1st level title (ones with `##`), however, the `Setting up and getting started` I left it uncapitalized because the guide name is not capitalized, i.e. this would be weird 🔽 (at least to me..)

> ## **Setting up and Getting Started**
>
> Refer to the guide [_Setting up and getting started_](SettingUp.md).

but if you think the above is okay then I can capitalize the title back!

Reply:
Oh ok, I think this classification of what gets capitalised is fine. Should be consistent throughout though. 

You should just change the title of the Setting Up and Getting Started page.